indico/modules/events/abstracts/templates/display/_common.html

```diff
@@ -36,7 +36,8 @@
                            data-title="{% trans %}Submit new abstract{% endtrans %}"
                            data-href="{{ url_for('abstracts.submit', event) }}"
                            data-confirm-close-unsaved
-                           data-ajax-dialog>
+                           data-ajax-dialog
+                           href="#">
```

Comment:
I think any `a.i-button` elements may be suitable for being replaced with actual `<button>` tags.

Reply:
@bxo11 would you have time to look into this option? I think it's cleaner (also from a semantic point of view) than adding hrefs :)